main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
```

Comment:
Variable names are terrible. Also, could probably be a list or dictionary - a lot of repetition.  

```suggestion
// All the questions that the program asks the user
    noun = input("Choose a noun: ")
    plural_noun = input("Choose a plural noun: ")
    second_noun = input("Choose a noun: ")
    place = input("Name a place: ")
    adjective = input("Choose an adjective (Describing word): ")
    third_noun = input("Choose a noun: ")
```

Reply:
Something like...
```
word_dict = dict(
    noun=""
    plural_noun=""
    second_noun = ""
    place = ""
    adjective = ""
    third_noun = ""
)

for key in word_dict:
    word_dict[key] = input(f"Choose a {key}")
```